cmake/module/FindQt.cmake

```diff
@@ -56,11 +56,11 @@ else()
 endif()
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(Qt5
-  REQUIRED_VARS Qt5_DIR
-  VERSION_VAR Qt5_VERSION
+find_package_handle_standard_args(Qt
+  REQUIRED_VARS Qt${Qt_FIND_VERSION_MAJOR}_DIR
+  VERSION_VAR Qt${Qt_FIND_VERSION_MAJOR}_VERSION
```

Comment:
Do we need to keep the versions here?

Reply:
It logs the actual Qt version in the output:
```
-- Found Qt: /usr/lib/x86_64-linux-gnu/cmake/Qt5 (found suitable version "5.15.13", minimum required is "5.11.3") 
```
Otherwise, the output will be less informative:
```
-- Found Qt: /usr/lib/x86_64-linux-gnu/cmake/Qt5 (Required is at least version "5.11.3")
```